modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/constants/SearchAdminConstants.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.search.constants;
+
+/**
+ * @author Gustavo Lima
+ */
+public class SearchAdminConstants {
```

Comment:
i dont think i would do this... if i did add a constants class, i would probably add them to com.liferay.portal.search.admin.web.internal.constants, not portal-search-api

also, you only applied to SearchAdminDisplayContextBuilder, but left out view.jsp etc

unless BChan asked for constants, we can probably leave them out. this pr is pretty big already

Reply:
Yeah I will do that, thx!